oak/server/oak_runtime.cc

```diff
@@ -146,6 +146,12 @@ grpc::Status OakRuntime::Start() {
   {
     LOG(INFO) << "Calling Rust runtime";
     int32_t rust_check = add_magic_number(1000);
+
+    if (rust_check != 42) {
```

Comment:
Isn't this supposed to return 1000 + 42 = 1042? Otherwise, if you have changed the implementation, perhaps rename the function to something else?

Reply:
Sorry, its a bit misleading, currently the input is ignored